src/ajax/xhr.js

```diff
@@ -6,9 +6,7 @@ define( [
 "use strict";
 
 jQuery.ajaxSettings.xhr = function() {
-	try {
-		return new window.XMLHttpRequest();
-	} catch ( e ) {}
+	return new window.XMLHttpRequest();
```

Comment:
In environments like jsdom the user would need to ensure there's a fake XHR stub in `window` or jQuery will throw an error here. That's a bit of a different behavior than currently, where it returns `undefined`. Any attempt to use `$.ajax` will fail either way, but I wonder if anyone calls `$.ajaxSettings.xhr()` to detect whether XHR is enabled. Probably not?

Reply:
A call to `$.ajax` would fail anyway. Only difference is the exception might actually be a little more informative with this change. I'm pretty certain nobody actually calls `$.ajaxSettings.xhr()` as a means to control if there is an XHR implementation and the change does not preclude from overriding `$.ajaxSettings.xhr` if needs be.

It's a +1 for me.